update_docker_images_txt.py

```diff
@@ -0,0 +1,81 @@
+"""Update ./docker_images.txt."""
+
+import argparse
+import logging
+import os
+
+
+def main() -> None:
+    """Prep and execute Condor job.
+
+    Make scratch directory and condor file.
+    """
+    parser = argparse.ArgumentParser(
+        description=("Update the ./docker_images.txt"),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--action",
+        required=True,
+        choices=["add", "remove"],
+        help="What do you want to do with the docker images?",
+    )
+    parser.add_argument(
+        "--docker-tag",
+        required=True,
+        help="The docker image tag",
+    )
+    parser.add_argument(
+        "--dest-dir",
+        required=True,
```

Comment:
I think this shouldn't be required, but should be optional with a default of the empty string.